scripts/create-protection-market-convex.ts

```diff
@@ -0,0 +1,140 @@
+/**
+ * @notice Deploys a Convex trigger and creates a protection market
+ * @dev To deploy, you must define the CONVEX_POOL_ID environment variable and set it equal to one
+ * of the keys of the `pools` variable, such as usdc or dai. Sample deploy commands are below:
+ *
+ *     CONVEX_POOL_ID=28 yarn hardhat run scripts/create-protection-market-convex.ts
+ *     CONVEX_POOL_ID=16 yarn hardhat run scripts/create-protection-market-convex.ts --network mainnet
+ */
+
+import hre from 'hardhat';
+import '@nomiclabs/hardhat-ethers';
+import { Contract, ContractFactory, utils } from 'ethers';
+import chalk from 'chalk';
+import { getChainId, getContractAddress, getGasPrice, logSuccess, logFailure, findLog, waitForInput } from '../utils/utils'; // prettier-ignore
+import comptrollerAbi from '../abi/Comptroller.json';
+
+// Constants
+const { AddressZero } = hre.ethers.constants;
+const cozyMultisig = '0x1725d89c5cf12F1E9423Dc21FdadC81C491a868b';
+
+// STEP 0: ENVIRONMENT SETUP
+const provider = hre.ethers.provider;
+const signer = new hre.ethers.Wallet(process.env.PRIVATE_KEY as string, hre.ethers.provider);
+const chainId = getChainId(hre);
+
+// STEP 1: TRIGGER CONTRACT SETUP
+const platformIds = [10];
+// const recipient = signer.address;
+const recipient = '0xSetRecipientAddressHere'; // subsidy recipient
```

Comment:
Doesn't this need to be set?

Reply:
It's intentionally left unset in all deploy scripts to (1) help prevent accidental deploys and (2) ensure the deployer remembers to change the recipient address to the correct one before deploying